### DISCUSSION TITLE
Reset font to body font on new line in text editor

### DISCUSSION THREAD

**Discussion by francescopetrungaro** [danielsaidi/RichTextKit#209] [Q&A]
Hi there,
I’m working on a test editor and I’m trying to achieve a behavior similar to Apple’s Notes app. Specifically, when the user goes to a new line, I want the font to automatically reset to the body font.

I managed to implement something close to this, but I’ve noticed an issue: when I move the cursor and print the font attributes, they appear incorrect, even though the UI displays the correct font. Has anyone encountered a similar problem or knows how to fix this? Any guidance would be greatly appreciated!​

**Comment by danielsaidi** [comment]
Hi @fralondon83 

There seems to be some problems with how fonts are handled. I'm not working on this project atm, due to business prioritizations, but I will keep this in mind.
